### PR TITLE
Update for podman pause test

### PR DESCRIPTION
**Pull request by ypu**
Update the method to get ctr_id and enable three tests.

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

File: test/podman_pause.bats [hunk 1]
```diff
@@ -104,7 +101,7 @@ function teardown() {
     run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} ps -a --filter id=$ctr_id"
     echo "$output"
     [ "$status" -eq 0 ]
-    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} rm $ctr_id"
+    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} rm -f $ctr_id"
```
Comment (TomSweeneyRedHat) [inline test/podman_pause.bats]: Unless this is causing issues, I'd rather not remove the '-f' here.  I'd left it off initially to make test that remove would work without it.

**Comment by mheon** [comment]
Homu looks like a flake pulling images

**Comment by mheon** [comment]
@ypu Can you fix the problem @TomSweeneyRedHat found before we try and merge again?

File: test/podman_pause.bats (hunk 1 above)
Comment (ypu) [reply test/podman_pause.bats]: Hi @TomSweeneyRedHat 
Want to confirm with you about this. As the stop command will not success to stop the paused container, the container will be running after unpause. So from current code in the rm step, we must add "-f" to it.

If the expect result is the paused container can be stopped then seems we should update the status check for stop command from "[ "$status" -eq 1 ]" to " [ "$status" -eq 0 ]".

**Comment by ypu** [comment]
@mheon Sure, will confirm the steps with him and update the code.

File: test/podman_pause.bats (hunk 1 above)
Comment (TomSweeneyRedHat) [reply test/podman_pause.bats]: @ypu thanks for the background information.  I'm still puzzled why this test was working prior, you're right the rm without the -f should fail here as the container should be running and it should not be removed without the -f parameter.

I'm fine with this change as is, but think I'll tweak this test slightly after this is merged.

**Comment by TomSweeneyRedHat** [comment]
@rh-atomic-bot r+

**Comment by TomSweeneyRedHat** [comment]
Closing and reopening as I can't get the tests to retry nicely.

**Comment by TomSweeneyRedHat** [comment]
reopening

**Comment by mheon** [comment]
@rh-atomic-bot retry

**Comment by mheon** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**